### ISSUE TITLE
Release 2 | Build Schedule Remove VLJs

### ISSUE THREAD

**Issue by Chingujo** [department-of-veterans-affairs/caseflow#7605]
As a hearing coordinator, I want to be able to remove a Veteran Law Judge from the schedule to account for changes in VLJ availability.

### Acceptance Criteria
- [ ] Verify that there is a way for the user to remove a VLJ from the Build schedule UI 
- [ ] Verify that there is a confirmation when the action is completed

**Comment by Chingujo** [comment]
@MeredithStewart 

We talked about removing judges when they retire or leave, but we would want to add as well? Otherwise they would have to run the judge algorithm every time they added/removed dates?

**Comment by MeredithStewart** [comment]
@Chingujo yes, need ability to add VLJ to the schedule - this will happen as frequently as the remove functionality. The judge algorithm wouldn't be good to run for these -- it's designed for bulk, 1 x use prior to the quarter.

**Comment by MeredithStewart** [comment]
Updated user stories and acceptance criteria 11/15/18.

**Comment by laurenrussell** [comment]
https://va-gov.invisionapp.com/share/EPP1JSLWN48

**Comment by MeredithStewart** [comment]
Updated to add the functionality to remove the hearing day (with no Veterans selected).

**Comment by oscarAtNava** [comment]
Completed together with Add ticket.
